css-responsive_playground/css/styles.css

```diff
@@ -94,3 +94,39 @@ h1 {
 .box--five img {
   width: 100%;
 }
+
+/* :::::: MEDIA QUERIES :::::: */
+/* second challenge*/
+@media only screen and (min-width: 599px) {
```

Comment:
why the "only" i used without only and it worked. Curious what only means in this context